### DISCUSSION TITLE
Archive to multiple farmers

### DISCUSSION THREAD

**Discussion by VercauterenK** [ericaltendorf/plotman#425] [Q&A]
Hi,

I have an threadripper that makes 48 files in 7 hours now the archive function can't handle that speed since the external drives can only handle 120mb/s is there a way to sync multiple files at once but at diffrent farmers. I have 10 harverster that has multiple external hard drives. If it sync 1 to all then i would not have that problem.

**Comment by thomann061** [comment]
Do you use 1gigabit ethernet? Because top ethernet speeds of 125MB/s would be the bottleneck regardless of how many files are synced at once.

**Comment by VercauterenK** [comment]
I am using 1gigabit but the threadripper is a disaster it’s to fast.

**Reply by altendky** [reply]
I have never used it, but fyi, there is https://github.com/rjsears/chia_plot_manager.  More flexible archiving in plotman is in the works but will take time to develop.  May also want to just shuttle drives depending...  a hard drive in your hand has a little latency, but plenty of bandwidth.

**Reply by VercauterenK** [reply]
Thanks gonna try that, at the moment i am plotting with 6 amd 5900x and 2 threadrippers 64 core so making allot of plots. The 5900x sending it al to their own harvesters so they can keep up. But the threadrippers are a real pain in the ass.

**Comment by thomann061** [comment]
You maybe looking on adding 10gigabit hardware...

**Reply by altendky** [reply]
Or...  I think it is `bond`ing multiple gigabit.  Depends how far you are going, etc.
